tests/modbus_reader/operations.robot

```diff
@@ -10,8 +10,19 @@ Suite Setup     Set Child Device1
 Device should set supported operations for writing to registers and coils
     Cumulocity.Should Contain Supported Operations    c8y_SetRegister    c8y_SetCoil
 
-Device should write to a registers
-    ${operation}=    Cumulocity.Create Operation    {"c8y_SetRegister": { "input": false, "address": 1, "startBit": 0, "noBits": 16, "ipAddress": "simulator", "value": 23, "register": 3 }}    description=Write to register
+Device should write to a register using legacy format
+    [Documentation]    Test writing to register using legacy format with explicit parameters
+    ${operation}=    Cumulocity.Create Operation    {"c8y_SetRegister": { "input": false, "address": 1, "startBit": 0, "noBits": 16, "ipAddress": "simulator", "value": 23, "register": 3 }}    description=Write to register (legacy format)
```

Comment:
`(legacy format)` is a little misleading as I don't think we'll be removing this feature. We can called it `(explicit address/register)` instead.